doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -1039,6 +1039,11 @@ cJSON
 
 |no_changes_yet_note|
 
+IDE and tool support
+=====================
+
+* Removed :file:`scripts/tool-version-minimum.txt` because we only test with 1 toolchain version.
```

Comment:
```suggestion
* Removed :file:`scripts/tool-version-minimum.txt` because we only test with one toolchain version.
  The table in :ref:`required-tools` has been updated accordingly to no longer specify minimum tool versions.
```